Breadcrumbs

britthurley opened the pull request:
Work on the following cards:

https://trello.com/c/18KbbhI7/547-website-navigation
https://trello.com/c/Yd0ugl7I/527-publish-open-house-page-on-site-and-link-it-to-main-page-somehow

Commit:
Add menu system and change flow

Commit:
Add event calendar

Commit:
Add event calendar

P0NDER0SA commented on _data/i18n.yml:
```diff
@@ -33,9 +41,17 @@ fr:
   canada_ca: Government of Canada
```
Still some english in here :)

P0NDER0SA committed:
Update faq.md

P0NDER0SA reviewed (changes requested):
noticed some english in the french section.  i  can check out how the app looks once this deploys to our new int environment for a more thorough review !

P0NDER0SA committed:
Merge branch 'main' into breadcrumbs

Commit:
links to workshop added

Commit:
Changed Learn more to Dojo Experience

Commit:
Internationalization work

stephanieDG committed:
Update i18n.yml

translated french terms.

stephanieDG committed:
Update apprendre.md

Commit:
fixed to trello card: https://trello.com/c/7kCp7goe/660-issue-in-breadcrumbs-branch-w-internationalization

Commit:
Fixed home links on the banner

Commit:
Add open house to sidebar, fix header home link

Commit:
fixed issues with workshop page

stephanieDG committed:
Update portesouvertes.md

Changed porte ouverte->atelier, removed last question and reference to kumospace instead of teamflow

stephanieDG committed:
Update openhouse.md

open house-> workshop

P0NDER0SA reviewed (approved):
we're good!
